haskell/nettest/Nettest.hs

```diff
@@ -50,10 +92,186 @@ scTransferScenario constructInitialStorage contract_ = uncapsNettest $ do
       (#to_ .! to)
       (#amount .! value)
 
-    -- In this transfer sender and @from@ address are always equal.
-    callTransfer from to value =
-      callFrom (AddressResolved from) sc (ep "transfer") (tp from to value)
+    callTransferWithOperator op from to value =
+      callFrom (AddressResolved op) sc (ep "transfer") (tp from to value)
+
+    callTransfer = join callTransferWithOperator
+
+    callTransfers
+      :: [("from_" :! Address, [("to_" :! Address, "amount" :! Natural)])]
+      -> capsM ()
+    callTransfers = mapM_ $ \(from@(arg #from_ -> from_), destinations) ->
+      callFrom
+        (AddressResolved from_)
+        sc
+        (ep "transfer")
+        (constructTransfersFromSender from destinations)
+
+    configureMinter :: Address -> Address -> Maybe Natural -> Natural -> capsM ()
+    configureMinter from for expectedAllowance newAllowance =
+      callFrom (AddressResolved from) sc (ep "configure_minter") (#minter .! for, (#current_minting_allowance .! expectedAllowance, #new_minting_allowance .! newAllowance))
```

Comment:
Some lines here are :so-big:, this one is 173 characters long, it's not ok.

Reply:
:bruh: